sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md

```diff
@@ -0,0 +1,398 @@
+# Analyze a document with add-on capabilities
+
+This sample demonstrates how to analyze document with add-on capabilities, using a document as an example. For more information about add-on capabilities, see the [documentation][docint_addon].
+
+To get started you'll need a Cognitive Services resource or a Document Intelligence resource. See [README][README] for prerequisites and instructions.
+
+Add-on capabilities are available within all models except for the Business card model. This sample uses Layout model ("prebuilt-layout") to demonstrate. When analyzing a document, you can specify a list of optional [analysis features][sdk_docfeature] to enable certain add-on capabilities.
+
+The following capabilities are free:
+- Barcodes
+- Languages
+- KeyValuePairs
+
+The following capabilities will incur additional charges, see [pricing][docint_pricing]:
+- Formulas
+- OcrHighResolution
+- FontStyling
+
+## Creating a `DocumentIntelligenceClient`
+
+To create a new `DocumentIntelligenceClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Document Intelligence API key credential by creating an `AzureKeyCredential` object that, if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateDocumentIntelligenceClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var client = new DocumentIntelligenceClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## High resolution extraction
+To extract content from a given file at a URI with improved quality through the add-on high resolution capability, use the `AnalyzeDocumentAsync` method and specify `DocumentAnalysisFeature.OcrHighResolution` as the analysis features. The returned value is an `AnalyzeResult` object containing data about the submitted document.

```

Comment:
I can understand other capabilities, but is there any point in having high resolution extraction as a sample if it doesn't change how we parse the output? I think we don't need to have a sample for every single add-on capability if we don't have anything special to show. Customers can figure out how to pass the `DocumentAnalysisFeature.OcrHighResolution` option with other samples.

Reply:
You're absolutely right. The output will maintain the same structure with this capability. However, our goal is to provide a sample corresponding to each supported capability. If the sample code is kind of duplicate with existing ones, can we still keep the description part but remove the code snippet?

Reply:
Removed the code snippets for high resolution.